server/webapp/WEB-INF/rails.new/app/views/admin/materials/pluggable_scm/show_existing.html.erb

```diff
@@ -0,0 +1,46 @@
+<%= render(:partial => "shared/flash_message") %>
+<%= render :partial => "shared/config_save_actions" -%>
+
+<%= form_for(@material, :as => :material, :url => admin_pluggable_scm_choose_existing_path, :html => {"novalidate"=> "novalidate", :id=> "material_form", :name=> "material_form", :method => "POST", :onsubmit => "return AjaxForm.jquery_ajax_submit(this);", :class => "popup_form"}) do |f| %>
+    <%= md5_field %>
+
+    <div class="form_content exec_task_editor">
+        <%= render :partial => "admin/shared/global_errors.html", :locals => {:scope => {}} -%>
+        <div id="material">
+            <%= render :partial => 'shared/form_required_message.html', :locals => {:scope => {}} %>
+
+            <%- if @scms.empty? -%>
+                <p class="warnings">No SCMs found. Please add a SCM first.</p>
+            <%- end -%>
+
+            <div class="fieldset">
```

Comment:
We talked about this, right? About how this allows any one to edit every SCM configured in the system. Can we (easily) restrict that? So that edit can be done only by admins? I ask because it's easier to go back and give permissions to everyone, than to take it back (should we choose that the behavior is wrong).